### ISSUE TITLE
Contributing to Accessibility Datasets: Reflections on Sharing Study Data by Blind People

### ISSUE THREAD

**Issue by XiyueWang-Miraikan** [Wotipati/Blind_Accessibility_Papers#587]
### Links
- Paper: https://dl.acm.org/doi/10.1145/3544548.3581337

### Abstract
To ensure that AI-infused systems work for disabled people, we need to bring accessibility datasets sourced from this community in the development lifecycle. However, there are many ethical and privacy concerns limiting greater data inclusion, making such datasets not readily available. We present a pair of studies where 13 blind participants engage in data capturing activities and reflect with and without probing on various factors that influence their decision to share their data via an AI dataset. We see how different factors influence blind participants’ willingness to share study data as they assess risk-benefit tradeoffs. The majority support sharing of their data to improve technology but also express concerns over commercial use, associated metadata, and the lack of transparency about the impact of their data. These insights have implications for the development of responsible practices for stewarding accessibility datasets, and can contribute to broader discussions in this area.

<img width="750" alt="スクリーンショット 2023-08-17 14 46 43" src="https://github.com/Wotipati/Blind_Accessibility_Papers/assets/91875613/6d8bcf5c-8f62-4b24-84a5-bb1deaada74b">
<img width="750" alt="スクリーンショット 2023-08-17 14 47 42" src="https://github.com/Wotipati/Blind_Accessibility_Papers/assets/91875613/fa1d8e52-ed07-40d2-b50f-fb6b6daed73e">

- A research that invites participants to take pictures and create a dataset, and discuss the participants's opinion about should the pictures and metadata be shared and how they should be shared.
